### PR TITLE
Fix record builder for TDH_INTYPE_SID

### PR DESCRIPTION
**Pull request by jrave**

### CODE REVIEW

File: tests/ManagedETWTests/EtwTestsCS.csproj [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>EtwTestsCS</RootNamespace>
     <AssemblyName>EtwTestsCS</AssemblyName>
-    <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
```
Comment (swannman) [inline tests/ManagedETWTests/EtwTestsCS.csproj]: Is this change required?

Comment (jrave) [reply tests/ManagedETWTests/EtwTestsCS.csproj]: No, removed it.
